tecken/base/decorators.py

```diff
@@ -117,7 +116,7 @@ def api_require_http_methods(request_method_list):
     """
 
     def decorator(func):
-        @wraps(func, assigned=available_attrs(func))
```

Comment:
`available_attrs` is removed from Django. `assigned` takes what `available_attrs(func)` returned by default so we can remove this argument.